check_if_image_instrumentable.sh

```diff
@@ -86,9 +86,12 @@ fi
 
 
 # supported libc versions
-supported_libc=(glibc-2.26-33.amzn2.0.1.x86_64.rpm* glibc-2.26-33.amzn2.0.2.x86_64.rpm* libc-bin_2.19-0ubuntu6.13_amd64.deb* libc-bin_2.19-0ubuntu6.14_amd64.deb* libc-bin_2.19-0ubuntu6.15_amd64.deb* libc-bin_2.19-0ubuntu6.6_amd64.deb* libc-bin_2.19-0ubuntu6.7_amd64.deb* libc-bin_2.19-0ubuntu6.9_amd64.deb* libc-bin_2.19-18+deb8u10_amd64.deb* libc-bin_2.19-18+deb8u6_amd64.deb* libc-bin_2.19-18+deb8u7_amd64.deb* libc-bin_2.23-0ubuntu10_amd64.deb* libc-bin_2.23-0ubuntu11_amd64.deb* libc-bin_2.24-11+deb9u1_amd64.deb* libc-bin_2.24-11+deb9u3_amd64.deb* libc-bin_2.24-11+deb9u4_amd64.deb* libc-bin_2.27-3ubuntu1_amd64.deb* libc-bin_2.27-3ubuntu1.4_amd64.deb* musl-1.1.15-r8.apk* musl-1.1.16-r14.apk* musl-1.1.16-r15.apk* musl-1.1.18-r2.apk* musl-1.1.18-r3.apk* musl-1.1.19-r10.apk* musl-1.1.20-r3.apk* musl-1.1.20-r4.apk* musl-1.1.22-r3.apk* musl-1.1.24-r3.apk* musl-1.1.24-r9.apk* musl-1.1.24-r10.apk*)
+supported_libc=(glibc-2.26-33.amzn2.0.1.x86_64.rpm* glibc-2.26-33.amzn2.0.2.x86_64.rpm* libc-bin_2.19-0ubuntu6.13_amd64.deb* libc-bin_2.19-0ubuntu6.14_amd64.deb* libc-bin_2.19-0ubuntu6.15_amd64.deb* libc-bin_2.19-0ubuntu6.6_amd64.deb* libc-bin_2.19-0ubuntu6.7_amd64.deb* libc-bin_2.19-0ubuntu6.9_amd64.deb* libc-bin_2.19-18+deb8u10_amd64.deb* libc-bin_2.19-18+deb8u6_amd64.deb* libc-bin_2.19-18+deb8u7_amd64.deb* libc-bin_2.23-0ubuntu10_amd64.deb* libc-bin_2.23-0ubuntu11_amd64.deb* libc-bin_2.24-11+deb9u1_amd64.deb* libc-bin_2.24-11+deb9u3_amd64.deb* libc-bin_2.24-11+deb9u4_amd64.deb* libc-bin_2.27-3ubuntu1_amd64.deb* libc-bin_2.27-3ubuntu1.4_amd64.deb* musl-1.1.15-r8.apk* musl-1.1.16-r14.apk* musl-1.1.16-r15.apk* musl-1.1.18-r2.apk* musl-1.1.18-r3.apk* musl-1.1.19-r10.apk* musl-1.1.20-r3.apk* musl-1.1.20-r4.apk* musl-1.1.22-r3.apk* musl-1.1.24-r3.apk* musl-1.1.24-r9.apk* musl-1.1.24-r10.apk* 
```

Comment:
This is incorrect. The supported list of glibcs should not have the entries for instrumented version of glibcs (those having LI and QCRS suffixes)

Reply:
Done!